db/migrate/20100525094200_simplify_parameters.rb

```diff
@@ -1,11 +1,11 @@
 class SimplifyParameters < ActiveRecord::Migration[4.2]
   def up
-    remove_index  :parameters, [:host_id,      :type] if index_exists? :parameters, :host_id
-    remove_index  :parameters, [:hostgroup_id, :type] if index_exists? :parameters, :hostgroup_id
-    remove_index  :parameters, [:domain_id,    :type] if index_exists? :parameters, :domain_id
+    remove_index  :parameters, [:host_id, :type] if index_exists? :parameters, [:host_id, :type]
+    remove_index  :parameters, [:hostgroup_id, :type] if index_exists? :parameters, [:hostgroup_id, :type]
+    remove_index  :parameters, [:domain_id, :type] if index_exists? :parameters, [:domain_id, :type]
 
     rename_column :parameters, :host_id, :reference_id if column_exists? :parameters, :host_id
-    add_index     :parameters, [:reference_id, :type] if index_exists? :parameters, :reference_id
+    add_index     :parameters, [:reference_id, :type]
 
     Parameter.reset_column_information
```

Comment:
This appears like change in already released migration from 2010. We must be absolutely sure there are not side effects. And I think there can be one, I suggest to create a copy of these statements in a new migration (we can keep those changed but the copy will make sure the same is executed for upgraded users).

Reply:
tbh i'm not sure we need this index at all, we already have index on `["type", "reference_id", "name"]` and I don't see any sense in looking for a parameter with just reference id and no type.

Reply:
I'll just revert the change to the migration and clean up any leftover indices in my migration.